game.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body onmousedown="showCoords(event)">
+    <canvas id="main" width="360" height="360"></canvas>
+    <script src = "game.js">
```

Comment:
put the script tag at the end of dom. (before closing body tag)

Reply:
and find out why we need to do that

Reply:
done